notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb

```diff
@@ -0,0 +1,922 @@
+{
```

Comment:
<p>Global comment - all of your links take the user out of the notebook. I wonder if there is away you can have the links open in a new tab. I made a similar comment to Polong recently on his notebook. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Good point! Fixed!